functional-tests/src/test/java/com/redhat/che/functional/tests/BayesianPomXml.java

```diff
@@ -65,4 +67,19 @@ protected void removeDependency() {
     editor.deleteCurrentLine();
     editor.waitTabFileWithSavedStatus(PROJECT_FILE);
   }
+
+  @Override // when bug is fixed remove this method
+  protected void editorCheckBayesianError() {
+    editor.setCursorToLine(POM_EXPECTED_ERROR_LINE);
+    editor.moveCursorToText(POM_EXPECTED_ERROR_TEXT);
+    try {
+      editor.waitTextInToolTipPopup(ERROR_MESSAGE);
+    } catch (TimeoutException e) {
+      if (getTestApiEndpointUrlProvider().get().getHost().equals(CHE_PROD_URL)) {
+        throw new SkipException(
+            "Skipping test for prod - known issue: https://github.com/redhat-developer/rh-che/issues/1198.");
```

Comment:
No need to skip tests on prod since the issues  https://github.com/redhat-developer/rh-che/issues/1198 / https://github.com/openshiftio/openshift.io/issues/4689  are fixed

Reply:
Thank you for your information. As far as I tried, bayesian is not working on reopening the file. I've created issue for that: https://github.com/redhat-developer/rh-che/issues/1256 I'll change it in that class.